src/ovs/ovs_control.cpp

```diff
@@ -0,0 +1,1503 @@
+// Copyright 2019 The Alcor Authors.
```

Comment:
this OVS file will likely need the licensing format per team discussion.

Reply:
// Copyright (c) 2008-2017, 2019 Nicira, Inc.
// Copyright 2019 The Alcor Authors - file modified.

two lines added in the beginning of ovs_control.h and ovs_control.cpp